packages/stark-ui/src/modules/app-footer/components/app-footer.component.html

```diff
@@ -0,0 +1,25 @@
+<footer class="stark-app-footer-container">
+	<span translate>STARK.APP_FOOTER.COPYRIGHT</span>
+		&copy; 
+	<span>{{ copyrightPeriod }}</span>
+	<span class="bullet" *ngIf="legalInfoUrl">
+		&bull; 
+	</span>
+	<a *ngIf="legalInfoUrl"
+	href="{{ legalInfoUrl }}">
```

Comment:
Can you add the `target` and `rel` attributes to this anchor as in the old Star?